lib/internal/webstreams/transformstream.js

```diff
@@ -368,6 +368,7 @@ function initializeTransformStream(
     close() {
       return transformStreamDefaultSinkCloseAlgorithm(stream);
     },
+    __proto__: null
```

Comment:
nit: can you put the proto at the top of the file? That seems to be a more common way of using `__proto__`.

Reply:
> the top of the file

Did you mean the top of the object definition?

Reply:
Oh yes, sorry, I need more coffee today

Reply:
Ah sure 😄! this is resolved have put it on top